src/components/Navigation.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 
 const Navigation = () => {
   return (
     <div>
       <div className="App">
         <h1>React Router Mini</h1>
         <div>
-          <a href="">Home</a>
+          <Link to="/Home">Home</Link>
```

Comment:
👍  Links do not need to be in their individual divs. Just remember that at the end of the day, they are just anchor tags